ladybug_comfort/collection/pmv.py

```diff
@@ -157,6 +158,78 @@ def __init__(self, air_temperature, rel_humidity,
+
+        Args:
+            epw: A ladybug EPW object from which the UTCI object will be created.
+            include_wind: Set to True to include the EPW wind speed in the calculation.
+                Setting to False will assume a condition that is shielded from wind
+                where the human expereinces a very low wind speed of 0.1 m/s.
+                Default is True to include wind.
+            include_sun: Set to True to include the mean radiant temperature (MRT) delta
+                from both shortwave solar falling directly on people and long wave radiant
+                exchange with the sky. Setting to False will assume a shaded condition
+                with MRT being equal to the EPW dry bulb temperature. When set to True,
+                this calculation will assume no surrounding shade context, standing human
+                geometry, and a solar horizontal angle relative to front of person (SHARP)
+                of 135 degrees. A SHARP of 135 essentially assumes that a person typically
+                faces their side or back to the sun to avoid glare.
+                Default is True to include sun.
+            met_rate: Data Collection of metabolic rate in met or a single
+                metabolic rate value to be used for the whole analysis. If None,
+                default is set to 1.1 met (for seated, typing).
+            clo_value: Data Collection of clothing values rate in clo or a single
+                clothing value to be used for the whole analysis. If None, default is
+                set to 0.7 clo (for long sleeve shirt and pants).
+            external_work: Data Collection of external work in met or a single
+                external work value to be used for the whole analysis. If None,
+                default is set to 0 met.
+            pmv_parameter: Optional PMVParameter object to specify parameters under
+                which conditions are considered acceptable. If None, default will
+                assume a PPD threshold of 10%, no absolute humidity constraints
+                and a still air threshold of 0.1 m/s.
+
+        Returns:
+            An object with data collections of the PMV results as properties.
+
+        Usage:
+
+            .. code-block:: python
+
+                from ladybug.epw import EPW
+                from ladybug_comfort.collection.pmv import PMV
+
+                epw_file_path = './tests/epw/chicago.epw'
+                epw = EPW(epw_file_path)
+                pmv = PMV.from_epw(epw, include_wind=True, include_sun=True)
+
+                # 12 values for the average SET in each month
+                print(pmv.standard_effective_temperature.average_monthly_per_hour().values)
+        """
+        # get wind and mrt inputs
+        if include_wind is True:
+            wind_speed = epw.wind_speed.duplicate()
+            for i, spd in enumerate(wind_speed):
+                wind_speed[i] = spd * (2 / 3)
```

Comment:
what about not duplicating and do it in one go.
```python
wind_speed = [spd * (2 / 3) for spd in epw.wind_speed]
```

Also is the 2 / 3 for a standard conversion?

Reply:
If you read the docstring, you will see that the class expects to either have a DataCollection that is aligned with the other input data collections or it's a single float value to be used for all timesteps. What you have there would produce a list of 8760 values (not a DataCollection) so it would fail. So that is why I duplicate the existing EPW's DataCollection and reset the values.

Granted, we could write something a little more compact:
```
wind_speed = epw.wind_speed.duplicate()
wind_speed.values = [spd * (2 / 3) for i, spd in enumerate(wind_speed)]
```
... but the way I have it written right now is actually a bit faster for continuous DataCollections as you would expect from an EPW.

Reply:
Oh, and I forgot to say: 2/3 the wind speed at 10 meters is what was used as a standard when they built the UTCI model.  It's essentially the factor that converts from the air speed used in the Fiala multi-node comfort model and the polynomial approximation that UTCI uses .

I'll add an in-line comment about this.

Reply:
Just in case you want to change it you should remove enumerate.

```python
wind_speed = epw.wind_speed.duplicate()
wind_speed.values = [spd * (2 / 3) for spd in wind_speed]
```